src/plugin.js

```diff
@@ -188,77 +195,91 @@ const vr = function(options) {
       const face5 = [new THREE.Vector2(0.333, 1), new THREE.Vector2(0.666, 1), new THREE.Vector2(0.666, 0.5), new THREE.Vector2(0.333, 0.5)];
       const face6 = [new THREE.Vector2(0.666, 1), new THREE.Vector2(1, 0), new THREE.Vector2(1, 0.5), new THREE.Vector2(0.666, 0.5)];
 
-      movieGeometry.faceVertexUvs[0] = [];
+      player.vr.movieGeometry.faceVertexUvs[0] = [];
 
-      movieGeometry.faceVertexUvs[0][0] = [ face1[0], face1[1], face1[3] ];
-      movieGeometry.faceVertexUvs[0][1] = [ face1[1], face1[2], face1[3] ];
+      player.vr.movieGeometry.faceVertexUvs[0][0] = [ face1[0], face1[1], face1[3] ];
+      player.vr.movieGeometry.faceVertexUvs[0][1] = [ face1[1], face1[2], face1[3] ];
 
-      movieGeometry.faceVertexUvs[0][2] = [ face2[0], face2[1], face2[3] ];
-      movieGeometry.faceVertexUvs[0][3] = [ face2[1], face2[2], face2[3] ];
+      player.vr.movieGeometry.faceVertexUvs[0][2] = [ face2[0], face2[1], face2[3] ];
+      player.vr.movieGeometry.faceVertexUvs[0][3] = [ face2[1], face2[2], face2[3] ];
 
-      movieGeometry.faceVertexUvs[0][4] = [ face3[0], face3[1], face3[3] ];
-      movieGeometry.faceVertexUvs[0][5] = [ face3[1], face3[2], face3[3] ];
+      player.vr.movieGeometry.faceVertexUvs[0][4] = [ face3[0], face3[1], face3[3] ];
+      player.vr.movieGeometry.faceVertexUvs[0][5] = [ face3[1], face3[2], face3[3] ];
 
-      movieGeometry.faceVertexUvs[0][6] = [ face4[0], face4[1], face4[3] ];
-      movieGeometry.faceVertexUvs[0][7] = [ face4[1], face4[2], face4[3] ];
+      player.vr.movieGeometry.faceVertexUvs[0][6] = [ face4[0], face4[1], face4[3] ];
+      player.vr.movieGeometry.faceVertexUvs[0][7] = [ face4[1], face4[2], face4[3] ];
 
-      movieGeometry.faceVertexUvs[0][8] = [ face5[0], face5[1], face5[3] ];
-      movieGeometry.faceVertexUvs[0][9] = [ face5[1], face5[2], face5[3] ];
+      player.vr.movieGeometry.faceVertexUvs[0][8] = [ face5[0], face5[1], face5[3] ];
+      player.vr.movieGeometry.faceVertexUvs[0][9] = [ face5[1], face5[2], face5[3] ];
 
-      movieGeometry.faceVertexUvs[0][10] = [ face6[0], face6[1], face6[3] ];
-      movieGeometry.faceVertexUvs[0][11] = [ face6[1], face6[2], face6[3] ];
+      player.vr.movieGeometry.faceVertexUvs[0][10] = [ face6[0], face6[1], face6[3] ];
+      player.vr.movieGeometry.faceVertexUvs[0][11] = [ face6[1], face6[2], face6[3] ];
 
-      movieScreen = new THREE.Mesh(movieGeometry, movieMaterial);
-      movieScreen.position.set(position.x, position.y, position.z);
+      player.vr.movieScreen = new THREE.Mesh(player.vr.movieGeometry, player.vr.movieMaterial);
+      player.vr.movieScreen.position.set(position.x, position.y, position.z);
 
-      scene.add(movieScreen);
+      player.vr.scene.add(player.vr.movieScreen);
     }
 
-    currentProjection = projection;
+    player.vr.currentProjection = projection;
   }
 
-  function initScene() {
+  /* reset player.vr to a default un-initialized state */
+  player.vr.reset = function() {
```

Comment:
reset the vr plugin in-between inits, so that if the source is not 360 we don't mess with it.